iFixit/react-commerce
# Upgrade Strapi to fix dynamic zone bug

Closes #1818

## QA

1. Open [govinor preview](https://fix-strapi-dynamic-zone-bug.govinor.com/admin/content-manager/collectionType/api::page.page/1?plugins[i18n][locale]=en) for this branch
2. Check that the issue described by #1818 is no longer present
3. Since we're upgrading Strapi, let's also perform a quick smoke test of product, product list pages and store home. It should be sufficient to verify that they render correctly.
